app/src/main/java/com/nutomic/syncthingandroid/activities/FolderActivity.java

```diff
@@ -162,26 +168,40 @@ public void onCreate(Bundle savedInstanceState) {
             // Open keyboard on label view in edit mode.
             binding.label.requestFocus();
             binding.editIgnores.setEnabled(false);
+            setOverrideChangesContainerEnabled(false);
         }
         else {
             // Prepare edit mode.
             binding.id.clearFocus();
             binding.id.setFocusable(false);
             binding.id.setEnabled(false);
             binding.directoryTextView.setEnabled(false);
+
+            // overridable remotes button
+            setOverrideChangesContainerEnabled(
+                getIntent().hasExtra(EXTRA_FOLDER_OVERRIDABLE_CHANGES)
+                && getIntent().getBooleanExtra(EXTRA_FOLDER_OVERRIDABLE_CHANGES, false)
```

Comment:
I agree, was simply following the existing code... will remove

Reply:
Ah didn't notice that. Just in case: Following existing code is a good argument to me. As in if in the future you intentionally dont change something and don't want to change it, and I miss it's preexisting, feel free to just tell me that without changing anything.